color/src/lib.rs

```diff
@@ -141,6 +142,21 @@ impl AlphaColor<Srgb> {
     }
 }
 
+impl PremulColor<Srgb> {
+    /// Create a color from pre-multiplied 8-bit rgba values.
+    ///
+    /// Note: for conversion from the [`PremulRgba8`] type, just use the `From` trait.
+    pub const fn from_rgba8(r: u8, g: u8, b: u8, a: u8) -> Self {
+        let components = [
+            u8_to_f32(r as u32),
```

Comment:
If you like, you can change the signature of `u8_to_f32` to take a u8; the current signature is no doubt a leftover from code that was also unpacking 32 bit values. But that doesn't have to be this PR.

Reply:
Done, and that let it fit on a single line.